app/components/channel_loader.js

```diff
@@ -103,3 +126,15 @@ const getStyleSheet = makeStyleSheetFromTheme((theme) => {
         }
     };
 });
+
+function mapStateToProps(state, ownProps) {
```

Comment:
This should be in its own file

Reply:
I can move it to its own file, no problem